esp/services/ws_fs/ws_fsService.cpp

```diff
@@ -105,6 +105,28 @@ void CFileSprayEx::init(IPropertyTree *cfg, const char *process, const char *ser
     
     xpath.clear().appendf("Software/EspProcess[@name=\"%s\"]/EspService[@name=\"%s\"]/QueueLabel", process, service);
     cfg->getProp(xpath.str(), m_QueueLabel);
+    StringArray qlist;
+    getDFUServerQueueNameArray(NULL, NULL, qlist);
+    if (qlist.ordinality())
+    {
+        if (!m_QueueLabel.length())
+            m_QueueLabel.append(qlist.item(0));
```

Comment:
The logic here is: if no default queue is configured,  the 1st DfuServer queue will be used as the default. The default will be used only if a user does not specify DfuServer queue.